data-reporting-service/src/main/resources/db/rdb/002-create_nrt_organization-001.sql

```diff
@@ -0,0 +1,40 @@
+USE RDB;
+drop table if exists dbo.nrt_organization;
```

Comment:
This is to align with RDB table creation policy that's part of the ETL? As I understand, we will not drop tables in ODSE, but for RDB will we be dropping and recreating or altering?

Reply:
@upasanapattnaik-eq Yes. In the future if the main Dim/Fact or DM tables have to be updated for some reason we should come up with ALTER table scripts, instead of drop and create. Other supporting tables could still get dropped and re-created.